### DISCUSSION TITLE
Frigate+

### DISCUSSION THREAD

**Discussion by Dolf123687** [blakeblackshear/frigate#15279] [General]
Hi everyone, apologies if this has been mentioned, did a quick search and didn't see it so please delete if it has, but is it just me, or does submitting to frigate+ in the beta releases feel much more tedious?

I am a frigate+ subscriber, but I kinda prefered pre 0.15, having it's own section, I'm not suggestion it needs it's own section, maybe just a way to see the next unsubmitted snapshot after submitting a snapshot, without needing to go back and selecting the next one and then submitting it.

**Comment by hawkeye217** [comment]
Filter the Explore view by "Has Snapshot?" / "Yes" and "Submitted to Frigate+?" / "No". Then click on the first item in the grid, select the Thumbnail tab. Submit the snapshot to Frigate+, then use your arrow keys to move to the next item. We've designed that workflow to be much faster than previous versions.

**Reply by Dolf123687** [reply]
Thank you

**Comment by NickM-27** [comment]
You can do this, you just need to use the filters to filter to items with snapshot then you can just press the frigate+ button on any of them to upload them

**Reply by Dolf123687** [reply]
Thank you

**Comment by Dolf123687** [comment]
My mistake guys, apologies, works perfectly
